libs/utils/src/lib/services/broadcast-channel/broadcast-channel.service.ts

```diff
@@ -0,0 +1,95 @@
+import { isPlatformBrowser } from '@angular/common';
+import { Inject, Injectable, PLATFORM_ID } from '@angular/core';
+import { EMPTY, fromEvent, Observable } from 'rxjs';
+
+@Injectable({
+	providedIn: 'root',
+})
+export class BroadcastChannelService {
+	private broadcastChannel: Record<string, BroadcastChannel> = {};
+
+	constructor(@Inject(PLATFORM_ID) private platformId: string) {}
+
+	/**
+	 * initChannel
+	 *
+	 * The initChannel method initializes a new BroadcastChannel instance.
+	 *
+	 * @param channelName{string} - The name of the Broadcast Channel.
+	 */
+	public initChannel(channelName: string): void {
+		if (!channelName) {
+			console.warn('channelName is required');
```

Comment:
This should in my opinion return an error and not a warning.

The error texts also aren't really in line with how we do it in other packages, we usually start with the package name `NgxUtils:` and then a more human readable error.

Reply:
I'd suggest `NgxUtils: A BroadcastChannel was tried to be initialised without a name. No channel has been initialised, please provide a name for this specific channel.`